packages/@dcl/inspector/src/hooks/editor/useSave.ts

```diff
@@ -0,0 +1,17 @@
+import mitt from 'mitt'
+import { useState } from 'react'
+
+import { useSdk } from '../sdk/useSdk'
+
+type SaveEvent = { change: boolean }
+export const saveEvent = mitt<SaveEvent>()
+
+/* istanbul ignore next */
```

Comment:
this seems like a very trivial hook to test, why ignore it?

Reply:
copy/paste from another hook. Will add test to it 👌